config/locales/forms/en.yml

```diff
@@ -52,7 +52,7 @@ en:
     email:
       buttons:
         delete: Delete email address
-    example: 'Example:'
+    example: 'example:'
```

Comment:
I believe this labels is used with hint labels throughout the application and this change may impact those instances. Just want to confirm we're aware / okay with that.

Reply:
> I believe this labels is used with hint labels throughout the application and this change may impact those instances. Just want to confirm we're aware / okay with that.

@theabrad @eric-gade @jmhooper Thoughts on this question?

Reply:
We have a couple other "example" hints in the application (e.g. [phone input](https://github.com/18F/identity-idp/blob/346a7b38cc4817b3452d5e1c1fe241951a5e4f05/app/components/phone_input_component.html.erb#L40), [memorable date](https://github.com/18F/identity-idp/blob/346a7b38cc4817b3452d5e1c1fe241951a5e4f05/config/locales/in_person_proofing/en.yml#L72)), so the main reason for asking is about consistency, since I think we should be consistent with the casing throughout the application, and this should be reflected in [relevant design system guidance](https://www.figma.com/file/kXLUewllU5YoQ6xG2fhwOM/Login.gov-Design-System-(LGDS)%3A-Figma-Library?node-id=1431%3A969&t=XatbjRmQZOIQwwRy-4) as well as [Figma flow designs](https://www.figma.com/file/iyRj58lOlPguVrayfARm0H/Identity-Verification-(Approved-%7C-Internal)?node-id=597%3A918&t=KKNl294hffCA2vMF-4).

Could y'all confirm the change with UX folks and create tickets if we want to change the case?